src/Index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Rename this file to `index.jsx`.

Reply:
@john555 thanks for the pointer i have implemented the requested changes